src/sqsd.groovy

```diff
@@ -68,8 +68,8 @@ try {
         def messages = sqs.receiveMessage(receiveMessageRequest).getMessages()
         println "Received Messages : " + messages.size()
 
-        // Break when empty
-        if(messages.size() <= 0) break
+        // Break when empty if not running daemonized
+        if(messages.size() <= 0 && config.sqsd.run_daemonized < 1) break
```

Comment:
I don't have any experience with phusion's base image. One technical thing I don't like about it is they chose ubuntu 14.04 over debian jessie which is more lightweight. For a dev image, I don't really care too much about image size.

I prefer containers which allow me to configure them over ones that require extension for these sorts of use cases. So if there's an environment variable or CMD argument I could pass to get daemonization, I'd prefer that.